### ISSUE TITLE
Display issue

### ISSUE THREAD

**Issue by gergo85** [stechstudio/filament-impersonate#59]
![display-issue](https://github.com/stechstudio/filament-impersonate/assets/2959112/83d8de4f-aa2e-42c2-89cb-62baecd45a80)

**Comment by MarJose123** [comment]
> ![display-issue](https://user-images.githubusercontent.com/2959112/273149998-83d8de4f-aa2e-42c2-89cb-62baecd45a80.png)

duplicate issues, here are some temporary fixes #54

**Comment by gergo85** [comment]
Thank you for your help!

**Comment by aurawindsurfing** [comment]
same issue here, anyone figured out the fix, please?

**Comment by gergo85** [comment]
@aurawindsurfing My solution:

I added the following function in AdminPanelProvider.php:
```
public function boot(): void
{
    FilamentAsset::register([
        Css::make('custom-stylesheet', public_path('css/app/custom-stylesheet.css')),
    ]);
}
```
The content of custom-stylesheet.css is:
```
div.fi-layout > aside.fi-sidebar {
    padding-top: 0 !important;
}
```

**Comment by aurawindsurfing** [comment]
Hey @gergo85 

It does the job, not perfect bug good enough. Maybe lest add it to the description of the plugin, or you are waiting for some filament to commit your pull request?

**Comment by nlvrendell** [comment]
On v3.5 : This code solved my issue

aside.fi-sidebar > div.overflow-x-clip {
    overflow-x: clip;
    position: sticky;
    top: var(--impersonate-banner-height);
    z-index: 20;
}

div.fi-layout > aside.fi-sidebar {
    padding-top: 0 !important;
}

**Comment by V13Axel** [comment]
Should be fixed as of 3.6, just released
